### ISSUE TITLE
txt files are blank

### ISSUE THREAD

**Issue by alucardnoyuuutsu** [lennylxx/IG_tools#13]
I'm trying to convert .s files from Kara no Shoujo HD Remaster, but when I do the program just outputs blank .txt files. What's the issue? Thanks

**Comment by SurveyorCangMing** [comment]
Same issue with Flowers -Le volume sur printemps- (Spring). Here's my solution. and you can take a try.
Using `igscript -x  -p 0000.s 0000.txt` and `igscript -x  -c 0000.s 0000.txt new\0000.s`
  instead of `igscript  -p 0000.s 0000.txt` and `igscript  -c 0000.s 0000.txt new\0000.s`

**Comment by lennylxx** [comment]
Yeah, sometimes you need to put an option `-x` to decrypt the script file
